compose/common.yml

```diff
@@ -162,7 +162,7 @@ services:
 
   ogmios:
     <<: *logging
-    image: cardanosolutions/ogmios:v${OGMIOS_VERSION:-6.1.0}
+    image: cardanosolutions/ogmios:v${OGMIOS_VERSION:-6.2.0}
```

Comment:
📝 I'll need to build this one for ARM, too #1172. Node 8.9.1 is already there.

Reply:
Done:
* [cardano-node:8.9.1](https://github.com/input-output-hk/ogmios-tracker/pkgs/container/ogmios-tracker%2Fcardano-node/197101568?tag=8.9.1)
* [ogmios:v6.2.0](https://github.com/input-output-hk/ogmios-tracker/pkgs/container/ogmios-tracker%2Fogmios/197122358?tag=v6.2.0)